ls1intum/Artemis
# change text exercises student result ui style for unreferenced feedback

### Checklist

### Motivation and Context


This is a follow-up PR to #1447
### Description

Adjusted the feedback for missing elements UI in students result view
### Steps for Testing

- check the students result view of text exercises that have feedback for missing elements
- check that the score is rendered correctly for all modeling exercises assessment views (including sample submission view)

### Screenshots


![image](https://user-images.githubusercontent.com/50635005/82924025-0e209980-9f7c-11ea-8d74-1b7ec59cb265.png)

**update**: I changed the design to follow the color scheme for the different amount of points

![image](https://user-images.githubusercontent.com/50635005/83430800-94ceee00-a436-11ea-9ef2-7b6e0837068b.png)
